debug_toolbar/apps.py

```diff
@@ -21,6 +21,17 @@ def check_middleware(app_configs, **kwargs):
     gzip_index = None
     debug_toolbar_indexes = []
 
+    # If old style MIDDLEWARE_CLASSES is being used, report an error.
+    if not settings.MIDDLEWARE:
```

Comment:
`django.conf.settings.is_overridden("MIDDLEWARE_CLASSES")` should work better for this.